blamo/lib/main.dart

```diff
@@ -39,6 +57,9 @@ class HomePage extends StatelessWidget {
   }
 }
 
+/* This is where Users will be creating and filling out documents for logging
+* This second page will house the text fields and (presumably) imaging options
+*/
```

Comment:
Could you detail a bit more what this behavior would be? Is this ultimately going to be creating a new log and redirecting  to the log info page of the new doc?

Reply:
Yeah the idea behind the home page is a series of existing logs will appear in the white space, While the button in the bottom right will allow users to create a new log (the "second page" in this code is mostly a demonstration and can/should be removed in later implimentation) additionally, the drawer will provide easy familiar navigation between setting, export, etc. Activities/pages of the project